docs/GraphNode.fbmd

```diff
@@ -270,111 +270,116 @@ Returns the `perms` property for the page as an `array` if present. (Only availa
 <card>
 ## GraphAlbum Instance Methods {#album-instance-methods}
 
-The `GraphAlbum` collection represents an [Album](https://developers.facebook.com/docs/graph-api/reference/album) Graph node.
```

Comment:
I save the JSON returned from `whatever?fields=id&metadata=1` in a whatever.json file, and use it as an input for my simple generator file (not published because it fully of if else bugs @#$% codes :D

Any way, you can take a look about it in this [Gist](https://gist.github.com/devmsh/81363a68fb62ccc28fb5#file-graph-api-subclass-generator)

I hate the repetitive work, and you hate magic, so I do my own magic behind the seen :P